### PR TITLE
chore(server): update README file

### PR DESCRIPTION
**Pull request by shaojunda**

### CODE REVIEW

File: server/README.md [hunk 1]
```diff
@@ -1,3 +1,56 @@
...
+
+[CKB](https://github.com/nervosnetwork/ckb) block explorer server
+
+## Prerequisites
+
+- [postgresql](https://www.postgresql.org/) 9.4 and above
+- install secp256k1 (see [secp256k1](https://github.com/bitcoin-core/secp256k1.git) for more info)
+
+```shell
+  $ cd ckb-explorer/tmp && git clone https://github.com/bitcoin-core/secp256k1.git && cd secp256k1 && ./autogen.sh && ./configure --enable-module-recovery --enable-experimental --enable-module-ecdh && make && sudo make install && cd ../..
```
Comment (classicalliu) [inline server/README.md]: no `tmp` dir

Comment (shaojunda) [reply server/README.md]: fixed

File: server/README.md [hunk 2]
```diff
@@ -1,3 +1,56 @@
...
+
+[CKB](https://github.com/nervosnetwork/ckb) block explorer server
+
+## Prerequisites
+
+- [postgresql](https://www.postgresql.org/) 9.4 and above
+- install secp256k1 (see [secp256k1](https://github.com/bitcoin-core/secp256k1.git) for more info)
+
+```shell
+  $ git clone https://github.com/bitcoin-core/secp256k1.git && cd secp256k1 && ./autogen.sh && ./configure --enable-module-recovery --enable-experimental --enable-module-ecdh && make && sudo make install && cd ../..
```
Comment (classicalliu) [inline server/README.md]: `cd ..` is enough

Comment (shaojunda) [reply server/README.md]: Thank you, fixed.

**Pull request merged**